tests/integration/devtools/test_props_check.py

```diff
@@ -216,3 +218,5 @@ def display_content(pathname):
             assert dash_duo.get_logs(), tc
         else:
             assert dash_duo.get_logs() == [], tc
+
+        time.sleep(0.1)
```

Comment:
what effect does this sleep have, coming after all the assertions?

Reply:
This test was failing but I am not sure that it needs this.